doc: note how to install cert-manager (#239)
kubernetes-sigs/kueue: docs/setup/install.md

- A Kubernetes cluster with version 1.21 or newer is running. Learn how to [install the Kubernetes tools](https://kubernetes.io/docs/tasks/tools/).
- The SuspendJob [feature gate](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/) is enabled. In Kubernetes 1.22 or newer, the feature gate is enabled by default.
- The kubectl command-line tool has communication with your cluster.

> - The [cert-manager](https://github.com/cert-manager/cert-manager) has been installed in your Kubernetes cluster. Learn how to [install cert-manager](https://cert-manager.io/docs/installation/).

Reviewer: ```suggestion
- The [cert-manager](https://github.com/cert-manager/cert-manager) is installed in your cluster. Learn how to [install cert-manager](https://cert-manager.io/docs/installation/).
```

To uninstall Kueue, run the following command:

```sh

> $ make undeploy

Reviewer: We are not putting `$` in the rest of the documentation.

```suggestion
make undeploy
```
